frontend/pages/api/nextjs/index.ts

```diff
@@ -0,0 +1,7 @@
+import type { NextApiHandler } from 'next';
+
+const handler: NextApiHandler = async (req, res) => {
+   res.status(200).json({ version: '3.0' });
```

Comment:
I'm curious why we're setting the initial version to `3.0` instead of `1.0`. @danielbeardsley's [argued](https://github.com/iFixit/react-commerce/pull/1264#issuecomment-1385939176) against versioning at all, but if we are going to use versions, I feel like it's ok to start from 1.0 since this api is distinct from iFixit's php api.

Reply:
Don't pay attention to this, it's just a dummy endpoint that we'll use to test the path forwarding before we merge another PR with the real caching endpoints.